src/vendor/perfherder.js

```diff
@@ -256,16 +295,17 @@ const getData = async (condition) => {
 
   const results = await Promise.all(collated
     .map(async ([pushDate, repo, framework, rest]) => {
-      if (repo === true || framework === true) {
+      if (missing(repo) || missing(framework)) {
```

Comment:
The `extract` function now returns `null` for a missing expression rather than `true` because `null` is easier to filter out, and it makes more sense.  Plus, they both mean the same thing:  `true` as a filter is always true, `null` as a filter means no expression, which means no filtering, which is the same as using `true`.